spec/models/role_spec.rb

```diff
@@ -15,6 +15,51 @@
     end
   end
 
+  describe '#state' do
+    let(:user) { FactoryGirl.create(:user) }
+    let(:team) { FactoryGirl.create(:team) }
+
+    before do
+      subject
+    end
+
+    subject do
+      user.roles.create team: team, name: role_name
+    end
+
+    context 'when the user is added as a coach' do
+      let(:role_name) { 'coach' }
+
+      it 'has the pending state' do
+        expect(subject).to be_pending
+      end
+    end
+
+    context 'when the user is added as a student' do
+      let(:role_name) { 'student' }
+
+      it 'has the pending state' do
```

Comment:
no worries, the same thing has happened to me SO often. Maybe that's why I caught it so quickly ;)